epictrack-web/src/components/layout/Header/EnvironmentBanner.tsx

```diff
@@ -5,18 +5,17 @@ import { useAppDispatch } from "../../../hooks";
 import { envBanner } from "../../../styles/uiStateSlice";
 import InfoIcon from "../../../assets/images/infoIcon.svg";
 import { ETSubhead } from "../../shared";
-
+import { AppConfig } from "../../../config";
 const EnvironmentBanner = () => {
   const dispatch = useAppDispatch();
-  const host = window.location.hostname;
+  const env = AppConfig.environment;
   const isTestEnvironment =
-    host.indexOf("dev") !== -1 ||
-    host.indexOf("test") !== -1 ||
-    host.indexOf("demo") !== -1 ||
-    host.indexOf("localhost") !== -1;
+    env === "dev" || env === "test" || env === "demo" || env === "localhost";

```

Comment:
use includes instead of multiple ===

Reply:
Also, include the env in upper case in the text that we are showing. Like "You are using a TEST environment"